packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -254,6 +275,7 @@ const machineBehavior = {
       }
       // TODO add aggregate debt tracking at the vaultFactory level #4482
       // totalDebt = AmountMath.add(totalDebt, toMint);
+      facets.machine.notifyEcon();
```

Comment:
That term started here https://github.com/Agoric/agoric-sdk/commit/15322a1f65bb67e1865f8276600a76171696dc74#diff-0b7d727a28b84089ac4ebe9b76f8957bb71bfee261640828bad01c4189cbe73fR283

I kept it because it distinguishes between the "creator facet" of the contract and the "limited creator facet" that it can return. I think we should move from "this thing is for what consumer" to "what this thing does". (For the same reasons that "public notifiers" is better than "UI notifiers")